src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -12,6 +12,9 @@
     <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
     <NoWarn>$(NoWarn);NU5128</NoWarn> <!-- No Dependencies-->
     <PackageDescription>Provides runtime information required to resolve target framework, platform, and runtime specific implementations of .NETCore packages.</PackageDescription>
+    
+    <!-- When building from source, ensure the RID we're building for is part of the RID graph -->
+    <AdditionalRuntimeIdentifiers Condition="'$(DotNetBuildFromSource)' == 'true'">$(AdditionalRuntimeIdentifiers);$(OutputRID)</AdditionalRuntimeIdentifiers>
```

Comment:
It's good we're appending the `OutputRID` at the end here. The `AdditionalRuntimeIdentifiers` will be added in order, so we want them to be ordered from least to most specific.

Reply:
That shouldn’t matter since the only way to establish a parent RID is through the parent parameter.  New RIDs are added recursively following the rules in the runtime groups and parent.  When written the graph is sorted.